docs/getting_started/faq.rst

```diff
@@ -0,0 +1,127 @@
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+While pandas works extremely well on small datasets, as soon as you start working with 
+medium to large datasets that are more than a few GBs, pandas can become painfully 
+slow or run out of memory. This is because pandas is single-threaded. In other words, 
+you can only process your data with one core at a time. This approach does not scale to 
+larger data sets and adding more hardware does not lead to more performance gain. 
+
+The ``modin.pandas`` :doc:`DataFrame </supported_apis/dataframe_supported>` is a highly 
+scalable, parallel DataFrame. Modin transparently distributes the data and computation so 
+that you can continue using the same pandas API while being able to work with more data faster. 
+Modin lets you use all the CPU cores on your machine, and because it is lightweight, it 
+often has less memory overhead than pandas. See this :doc:`page </getting_started/pandas>` to 
+learn more about how Modin is different from pandas. 
+
+Why not just improve pandas?
+""""""""""""""""""""""""""""
+
+pandas is a massive community and well established codebase. Many of the issues
+we have identified and resolved with pandas are fundamental to its current
+implementation. While we would be happy to donate parts of Modin that
+make sense in pandas, many of these components would require significant (or
+total) redesign of the pandas architecture. Modin's architecture goes beyond
+pandas, which is why the pandas API is just a thin layer at the user level. To learn
+more about Modin's architecture, see the :doc:`architecture </developer/architecture>` documentation.
+
+How much faster can I go with Modin compared to pandas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Modin is designed to scale with the amount of hardware available.
+Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
+distributing the work across your entire machine. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
+efficiently to larger machines with more cores. We have several published papers_ that
+include performance results and comparisons against pandas.
+
+How much more data would I be able to process with Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+We have focused heavily on bridging the solutions between DataFrames for small 
+data (e.g. pandas) and large data. Often data scientists require different tools 
+for doing the same thing on different sizes of data. The DataFrame solutions that 
+exist for 1MB do not scale to hundreds of GBs, and the overheads of the large-scale solutions 
+are too costly for datasets in the 1KB range. With Modin, because of its light-weight, 
+robust, and scalable nature, you get a fast DataFrame at 1MB and hundreds of GBs.
```

Comment:
```suggestion
Often data scientists have to use different tools for operating on datasets of different sizes. This is not only because processing large dataframes is slow, but also pandas does not support working with dataframes that don't fit into the available memory. As a result, pandas workflows that work well for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably work with hundreds of GBs without worrying about substantial slowdown or memory errors. For more information, see :doc:`out-of-memory support <getting_started/out_of_core.rst>` for Modin.
```